stable/aws-for-fluent-bit/values.yaml

```
@@ -11,6 +11,13 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+podSecurityContext: {}
+  #runAsUser: 1000
+containerSecurityContext: {}
+  #capabilities: 
+  #  Drop:
+  #  - all
```

```suggestion
  #  - ALL
```

Must be uppercased. This is uppercase in PSA, kyverno, and the only spec that exists, despite some runtimes being lenient regarding case.